### PR TITLE
refactor(Features/ACF): extract FieldLoader and adjust FieldGroupComp…

### PR DESCRIPTION
**Pull request by domtra**
…oser and OptionPages

### CODE REVIEW

File: Features/Acf/FieldLoader.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Flynt\Features\Acf;
+
+use RecursiveDirectoryIterator;
+use Flynt\Utils\ArrayHelpers;
+use Flynt\Utils\StringHelpers;
+use Flynt\Utils\FileLoader;
```
Comment (usingthesystem) [inline Features/Acf/FieldLoader.php]: i think that it not used or?

File: Features/Acf/FieldLoader.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace Flynt\Features\Acf;
+
+use RecursiveDirectoryIterator;
```
Comment (usingthesystem) [inline Features/Acf/FieldLoader.php]: i think that it not used or?

File: Features/Acf/FieldLoader.php [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+        foreach ($subFields as $subField) {
+            if (is_string($subField)) {
+                continue;
+            }
+            if (!array_key_exists('name', $subField)) {
+                trigger_error('[ACF] Name is missing in Sub Field while adding Filter: ' . $parentFilterName, E_USER_WARNING);
+                continue;
+            }
+            $subFieldName = ucfirst($subField['name']);
+            $subFilterName = $parentFilterName . "/{$subFieldName}";
```
Comment (usingthesystem) [inline Features/Acf/FieldLoader.php]: How about writing it like this?
$subFilterName = "{$parentFilterName}/{$subFieldName}";

Comment (domtra) [inline Features/Acf/FieldLoader.php]: yeah, i just copied and pasted this. but you are right.

**Pull request merged**